### DISCUSSION TITLE
Add turn arc to the ND

### DISCUSSION THREAD

**Discussion by fafaldo** [magknight/787aviatorsTracker#692] [Completed things] [answered]
Now that you guys are able to do more custom drawing over the ND, maybe you could add this turn/position arc that is found in most Boeing planes as well? Should be probably calculated without the need for custom FMS.

![image](https://user-images.githubusercontent.com/63355317/153430366-dbb9c750-3829-4a84-9961-be01f16d4450.png)

**Comment by aeroniemi** [comment, marked as answer]
Added in 1.7.28
